transformer_xaod.py

```diff
@@ -97,10 +103,21 @@ def callback(channel, method, properties, body):
                 servicex.put_file_complete(file_path=_file_path, file_id=_file_id,
                                         status='failure', num_messages=0, total_time=0,
                                         total_events=0, total_bytes=0)
+
+                servicex.post_status_update(file_id=_file_id,
+                                            status_code="failure",
+                                            info= " error: " + str(error)[0:1024])
```

Comment:
Just double checking here - will `error` contains a traceback?

Reply:
Just caught this - and up to 1024 - why is that a limit? Some of these can be quite deep, and it would be too bad to lose them.

Reply:
Same thing here - just want to catch enough to drive deeper investigation